src/app/item-card/item-card.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, Input, Output, EventEmitter } from "@angular/core";
+import { Item } from "../model/item.interface";
+
+@Component({
+  selector: "app-item-card",
+  templateUrl: "./item-card.component.html",
+  styleUrls: ["./item-card.component.scss"],
+})
+export class ItemCardComponent {
+  private maxDescriptionLength: number = 150;
+  private maxTitleLength: number = 14;
+  public itemData!: Item;
+  public description!: string;
+  public title!: string;
```

Comment:
Once you change the way to "cut" the length on description and title you won't need this anymore.